packages/app/components/Nation/NewNation/NewNation.tsx

```diff
@@ -0,0 +1,198 @@
+import * as React from 'react';
+import arrayMutators from 'final-form-arrays';
+import { Button, Form as SUIForm, Header, TextArea } from 'semantic-ui-react';
+import { FieldArray } from 'react-final-form-arrays';
+import { Form, FormRenderProps } from 'react-final-form';
+import * as yup from 'yup';
+
+import BackButton from '../../LeftPane/Pages/BackButton';
+import Checkbox from './Checkbox';
+import ColorPicker from './ColorPicker';
+import Field from './Field';
+import withHandleSubmit, {
+  WithHandleSubmitProps
+} from './decorators/withHandleSubmit';
+
+// TODO This should be defined with the model
+const validatePoliticalEntity = yup.object().shape({
+  color: yup
+    .string()
+    .matches(/^#([a-f0-9]{3,4}|[a-f0-9]{4}(?:[a-f0-9]{2}){1,2})\b$/i), // Color regex, e.g. #fa32be
+  description: yup.string(),
+  control_type: yup
+    .string()
+    .oneOf(['CC', 'DT'])
+    .required(),
+  links: yup.array().of(
+    yup
+      .string()
+      .url()
+      .required()
+  ),
+  name: yup
+    .string()
+    .max(100)
+    .required(),
+  references: yup
```

Comment:
Yes, didn't add it on purpose because I wanted to make a decision on https://github.com/chronoscio/backend/issues/28 first.

If we go the regex way, I'll add it to the frontend too.
If we go the new Reference model way, we define fields in the new model and I'll add validation on those fields.